docs/migration/test-cloud/index.md

```diff
@@ -1,13 +1,94 @@
 ---
-title: "Test Cloud Migration"
-author: joemellin
-ms.author: joemel
-ms.date: 04/10/2017
+title: Test Cloud Migration Guide
+description: "Xamarin Test Cloud to AppCenter test tranition guide. Relevant to Xamarin Test Cloud subscribers that will be automatically upgraded from Xamarin Test Cloud to Visual Studio App Center on March 31, 2018"
+keywords: testcloud
+author: glwest
```

Comment:
Is your Github ID the same as your Microsoft Alias?  Author is your github ID and ms.author is your microsoft alias.